Allow for polling of non namespaced resources (#87)
razee-io/Watch-keeper: README.md

1. Labeling namespaces, especially using the detail or debug level collections, can gather much more data than anticipated resulting in delays in data reporting.
1. Similarly,  delays can occur when reporting on a namespace with lots of resources (> thousand).

### Non Namespaced Resources

> In order to avoid having to label each individual non namespaced resource (eg. nodes, namespaces, customresourcedefinitions),

Reviewer: ```suggestion
In order to avoid having to label each individual non-namespaced resource (eg. nodes, namespaces, customresourcedefinitions),
```

we allow polling of all non namespaced resource. This mechanism is similar to how our namespace resource collection works,
where you can label a namespace and we collect all the resources within that namespace for you; you can think of this like you are labeling the `non-namespaced-resources` namespace.

> Also similar to how you can label a namespace, there may be resources that you dont want to collect (eg. storageclass), so

Reviewer: ```suggestion
Also similar to how you can label a namespace, there may be resources that you do not want to collect (eg. storageclass), so
```

you should use [white/black lists](#whiteblack-lists) to limit what is collected. Note: using the white/black list will

> effect all resources polled, namespaced and non namespace.

Reviewer: ```suggestion
affect all resources polled, namespaced and non namespace.
```
